SonarQube.TeamBuild.PostProcessor/MSBuildPostProcessor.cs

```diff
@@ -72,9 +72,14 @@ public bool Execute(string[] args, AnalysisConfig config, TeamBuildSettings sett
                 return false;
             }
 
-            // Handle code coverage reports
-            if (!this.codeCoverageProcessor.ProcessCoverageReports(config, settings, logger))
+            if (!this.codeCoverageProcessor.Initialise(config, settings, logger))
             {
+                // if initialisation fails (e.g. no VS is installed) just log a warning
+                logger.LogWarning(Resources.WARN_CannotProcessCoverage);
```

Comment:
How many warnings are produced if the code coverage report can't be processed? Two - one from this class and one from the "real" processor class? If so, do we need both?